src/main/java/dinkplugin/util/ItemUtils.java

```diff
@@ -118,7 +118,7 @@ public SerializedItemStack stackFromItem(ItemManager itemManager, Item item) {
     public SerializedItemStack stackFromItem(ItemManager itemManager, int id, int quantity) {
         ItemComposition composition = itemManager.getItemComposition(id);
         int price = getPrice(itemManager, id, composition);
-        return new SerializedItemStack(id, quantity, price, composition.getName());
+        return new SerializedItemStack(id, quantity, price, String.valueOf(composition.getMembersName()));
```

Comment:
Is `String.valueOf` just a safety-guard in case Runelite would return null here?

Is `getMembersName()` nullable as opposed to `getName()`, or is it just a nice just-in-case-fix either way?

Reply:
Yeah `String#valueOf` would convert null strings to `"null"` just in case RL yields null (shouldn't happen regardless)

`ItemComposition#getMembersName` is the exact same as `ItemComposition#getName` but it strips the ` (Members)` prefix from item names on F2P worlds